src/main/java/org/ays/emergency_application/model/request/EmergencyEvacuationApplicationRequest.java

```diff
@@ -115,9 +115,9 @@ private boolean isPhoneNumberMustNotBeSameOne() {
     @SuppressWarnings("This method is unused by the application directly but Spring is using it in the background.")
     private boolean isSourceCityAndDistrictDifferentFromTargetCityAndDistrict() {
 
-       if (this.sourceCity == null || this.sourceDistrict == null || this.targetCity == null || this.targetDistrict == null) {
+        if (StringUtils.isBlank(this.sourceCity) || StringUtils.isBlank(this.sourceDistrict) || StringUtils.isBlank(this.targetCity) || StringUtils.isBlank(this.targetDistrict)) {
             return true;
-       }
+        }
 
         if (!this.sourceCity.equalsIgnoreCase(this.targetCity)){
```

Comment:
Commitlemeden önce otomatik formatı açarsak bu tür unutulan ufak boşlukları düzeltebilir;
https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html
```suggestion
        if (!this.sourceCity.equalsIgnoreCase(this.targetCity)) {
```